exec_test.go

```diff
@@ -35,3 +36,43 @@ func ExampleCommandExists() {
 	// true
 	// false
 }
+
+func BenchmarkCommandExists(b *testing.B) {
+	cases := []struct {
+		Name    string
+		Command string
```

Comment:
```suggestion
		Command string
```

You can use the Command as sub-command name